tools/internal_ci/linux/grpc_distribtests_python.sh

```diff
@@ -50,6 +53,8 @@ cp -r artifacts/* input_artifacts/ || true
 # a better signal about which distribtest are affected by the currently broken artifact builds.
 tools/run_tests/task_runner.py -f distribtest linux python -j 12 -x distribtests/sponge_log.xml || FAILED="true"
 
+tools/internal_ci/helper_scripts/store_artifacts_from_moved_src_tree.sh
```

Comment:
This is a distribtest, do we need to preserve artifacts?

Reply:
see comment above.